### PR TITLE
Add Travis & Doxygen

### PR DESCRIPTION
**Pull request by szymonkaliski**
Ping @ladyada :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Adafruit APDS9960 Library [![Build Status](https://travis-ci.org/adafruit/Adafruit_APDS9960_Library.svg?branch=master)](https://travis-ci.org/adafruit/Adafruit_APDS9960_Library)
```
Comment (ladyada) [inline README.md]: change to travis-ci.com please :)

File: Adafruit_APDS9960.cpp [hunk 2]
```diff
@@ -476,102 +562,159 @@ uint16_t Adafruit_APDS9960::calculateLux(uint16_t r, uint16_t g, uint16_t b)
...
 }
 
-uint32_t Adafruit_APDS9960::read32(uint8_t reg)
-{
-	uint8_t ret[4];
-	this->read(reg, ret, 4);
-	
-	return (ret[0] << 24) | (ret[1] << 16) | (ret[2] << 8) | ret[3];
+/*!
+ *  @brief  Reads 32 bytes from specified register
```
Comment (ladyada) [inline Adafruit_APDS9960.cpp]: bits not bytes

File: Adafruit_APDS9960.cpp [hunk 3]
```diff
@@ -476,102 +562,159 @@ uint16_t Adafruit_APDS9960::calculateLux(uint16_t r, uint16_t g, uint16_t b)
...
 }
 
-uint8_t Adafruit_APDS9960::read8(byte reg)
-{
-	uint8_t ret;
-	this->read(reg, &ret, 1);
-	
-	return ret;
+/*!
+ *  @brief  Reads 8 bytes from specified register
```
Comment (ladyada) [inline Adafruit_APDS9960.cpp]: bits

File: Adafruit_APDS9960.cpp [hunk 4]
```diff
@@ -476,102 +562,159 @@ uint16_t Adafruit_APDS9960::calculateLux(uint16_t r, uint16_t g, uint16_t b)
...
 }
 
-uint16_t Adafruit_APDS9960::read16(uint8_t reg)
-{
-	uint8_t ret[2];
-	this->read(reg, ret, 2);
-	
-	return (ret[0] << 8) | ret[1];
+/*!
+ *  @brief  Reads 16 bytes from specified register
```
Comment (ladyada) [inline Adafruit_APDS9960.cpp]: ditto

File: Adafruit_APDS9960.cpp [hunk 5]
```diff
@@ -476,102 +562,159 @@ uint16_t Adafruit_APDS9960::calculateLux(uint16_t r, uint16_t g, uint16_t b)
...
+  uint8_t ret[2];
+  this->read(reg, ret, 2);
+
+  return (ret[0] << 8) | ret[1];
 }
 
-uint16_t Adafruit_APDS9960::read16R(uint8_t reg)
-{
+/*!
+ *  @brief  Reads 16 bytes from specified register
```
Comment (ladyada) [inline Adafruit_APDS9960.cpp]: ditto

**Review by ladyada — CHANGES_REQUESTED** [review]

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
shoot merged but the travis link is wrong - please fix

should be https://travis-ci.com/adafruit/Adafruit_APDS9960

**Comment by szymonkaliski** [comment]
→ https://github.com/adafruit/Adafruit_APDS9960/pull/16 :)

**Pull request merged**